src/app/app-interactive-map/store/interactive-map.effects.ts

```diff
@@ -121,7 +124,8 @@ export class InteractiveMapEffects {
       const model = action.payload;
       const {maps} = storeState.interactiveMap;
       const predicate = (mapItem: types.MapItem) =>
-        mapItem.model === model.name && mapItem.name === preferredMapsByModel[model.name];
+        (mapItem.model === model.name && mapItem.name === preferredMapsByModel[model.name]) ||
+        (mapItem.model === 'iJO1366' && mapItem.name === 'Central metabolism');
```

Comment:
Why did you have to hardcode it like this?

Reply:
To put by default the map central metabolism of iJO366